### ISSUE TITLE
[BUG] VoiceOver navigation on Menu needs improvement

### ISSUE THREAD

**Issue by chi-jyskebank-dk** [kirbydesign/designsystem#3780]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When a Kirby Menu is opened using VoiceOver, it can be navigated by swiping on the screen. However, the non-seeing user has no clue to where on the screen to swipe. Only if you swipe inside the menu, the menu-items are read out loud. But as all content you swipe past gains focus and is read out loud, it's hard to figure out what is in fact items of the menu. 

Probably even more so on an iPad where the user has even less clue where to put his finger, in order to operate to "local menu". 

## Describe how to reproduce the bug
I have disabled the DRB Feature Toggle "re2 - Account list card: Use micro front", in order to make sure that the Account List menu from the Relationsbank Hostapp is being used (and therefore the latest Kirby-version).

See attached screen recording for how to access the Account List, local menu.

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
When a local menu is open, it should be the only component that takes focus, in order to help the non-seeing user to operate it. 

## Add any screenshots

https://github.com/user-attachments/assets/67525edf-96ee-4a58-8af6-0e49b121f659

## Please complete the following information:
- OS: iOS 18.3
- Latest version of DRBTest App from Master

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
